src/mapValues.test.ts

```diff
@@ -1,5 +1,6 @@
 import { mapValues } from './mapValues';
 import { pipe } from './pipe';
+import { describe } from 'vitest';
```

Comment:
Hmmm... Why is this needed, we have global types for vitest, the file already used describe without it